src/NServiceBus.Core.Tests/Sagas/ComponentTests/SagaWithComplexTypeEntity.cs

```diff
@@ -0,0 +1,10 @@
+namespace NServiceBus.Persistence.ComponentTests
+{
+    using System.Collections.Generic;
+
+    public class SagaWithComplexTypeEntity : ContainSagaData
+    {
+        public string CorrelationProperty { get; set; }
+        public List<int> Ints { get; set; }
```

Comment:
You are right, but still we took the test suite from InMemPersistence and tweaked it. This test just proves that more complex things such as lists work as well. I think we need to address this further in the persistence alignment task. Ok if I ignore this comment?